Practica2/includes/src/pueblos/bd/Pueblo.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+class Pueblo
```

Comment:
Un poco raro que el pueblo tenga password. Realmente si queréis tener una jerarquía de clases. Usuario -> Pueblo, Usuario -> Empresa, etc. en Usuario estarían todos los atributos comunes (como nombre de usuario y contraseña) que utilizaríais para el login / registro, junto al rol que en este caso os serviría también para saber el usuario concreto (Pueblo, Empresa) que representa una fila en la tabla Usuarios.
Luego el resto de atributos estaría en otrasclases: Pueblo, Empresa, etc.